.github/workflows/main.yml

```diff
@@ -19,6 +19,10 @@ jobs:
         - os: windows-latest
           python: '3.12'
           toxenv: py
+        # typing
+        - os: ubuntu-latest
+          python: '3.8'
```

Comment:
I'm still bitter there's no way to tell GHA that "Hey, I have this list elsewhere of things I want to run against, can you just pick the 'earliest'/'oldest'/'smallest' so that when I update the list I don't have to update every goddamn reference to remove the oldest?"

Reply:
I'm a novice at best when it comes to GHA and just went off of what the rest of the workflow looked like, so I have no idea of the technology exists, lol. It is a bit annoying, but I figured if it does get refactored, this would temporarily showcase what the output would look like in CI for `pyright --verifytypes` and either help or hurt the case for switching to and/or including mypy :D.

Reply:
Oh, I'm just griping aloud, not hoping you would magically fix GitHub's product. 😂